deploy/docker/entrypoint.sh

```diff
@@ -25,4 +25,13 @@ if [ "$1" = "fluentd" ]; then
     fi
 fi
 
+# Install custom plugins if specified by user
+if [ -n "${ADDITIONAL_PLUGINS}" ]
+then
```

Comment:
Please follow the https://google.github.io/styleguide/shellguide.html#loops
`Put ; do and ; then on the same line as the while, for or if.`

It'll also be consistent throughout the file this way.